docs/source/getting_started/basic_functionality.rst

```diff
@@ -50,100 +58,38 @@ List
 ^^^^
 
 List returns an iterator of every resource in a collection.
-To list every material spec in the dataset ``standard_glasses_dataset``, use the command:
+To list every design space in the project ``uv_absorbing_glasses``, use the command:
 
 .. code-block:: python
 
-    standard_glasses_dataset.material_specs.list()
-
-You can instead broaden the scope of the search to an entire project.
-The command ``lens_project.material_specs.list()`` will list every material spec in every dataset that ``lens_project`` has read access to.
+    uv_absorbing_glasses.design_spaces.list()
 
 Updating
 --------
 
-The ``register`` command is also used to update an object. The following code creates and persists
+The ``update`` command updates an object. The following code creates and persists
 a process spec ``sintering`` to a dataset ``tungsten_dataset``, then updates it locally
 and persists that update.
 
 .. code-block:: python
 
     sintering = tungsten_dataset.register(ProcessSpec(name="Sinter a powder"))
     sintering.notes = "Forgot to add notes!"
-    tungsten_dataset.register(sintering)
+    tungsten_dataset.update(sintering)
 
 
 Deleting
 --------
 
-Deleting an object is a permanent action and cannot be undone, so do use caution when performing deletes!
-
-In order for a delete to be successful, the following circumstances must be true:
-
-1. The user has write access to the containing dataset
-2. Deleting the object won't invalidate or orphan other objects
-
-In the case that a delete fails, an error message will be returned indicating the point of failure.
-
-For example, any attempt to delete a ``MaterialSpec`` object that is referenced by a ``MaterialRun`` object will be unsuccessful because the ``MaterialRun`` would no longer have an associated ``MaterialSpec``.
-
-In this case, an error message will be returned with both the ``id`` of the referencing ``MaterialRun`` object *and* the ``id`` of its containing dataset.
-Should that ``MaterialRun`` itself be deleted, or associated with a different ``MaterialSpec`` object, the targeted ``MaterialSpec`` can then be deleted.
-
-Deleting uses this generalized approach:
-
-.. code-block:: python
-
-    dataset.object_type.delete(id)
-
-For example:
-
-.. code-block:: python
+Resources can generally be deleted with the ``delete`` command.
+However many resources are needed by other resources, and so deleting them is tricky.
```

Comment:
```suggestion
However some resources are needed by other resources, and deleting these intertwined objects is tricky.
```
I found this sentence a bit confusing. I'm not sure this suggestion is great, but maybe something along these lines.